ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -28,4 +28,6 @@ WM_WF:
   - WM
   is_canonical: true
   uses:
-  - supply_water_flowrate_sensor
\ No newline at end of file
+  - supply_water_flowrate_sensor
```

Comment:
flowrate sensor, remove supply water

Reply:
This is an existing filed in the canonical present in the git.

Reply:
> This is an existing filed in the canonical present in the git. Wouldn't it create conflict if we delete supply_water, elsewhere.

Reply:
created a new canonical 

WM_WVFM:
  description: "Water volume accumulator and flow rate measurement ."
  implements:
  - WM
  is_canonical: true
  uses:
  - flowrate_sensor
  - water_volume_accumulator